### PR TITLE
RPCSX-GPU & RPCSX-OS Merge Update

### PR DESCRIPTION
**Pull request by Darkhost1999**
Update the commands for running RPCSX
Add audio and nasm dependencies
Remove outdated information about Virtual HDD
Tease upcoming WSL information
Overall review and evaluate the information and ensure accuracy.

### CODE REVIEW

**Comment by Darkhost1999** [comment]
$ rubocop
Inspecting 8 files
........

8 files inspected, no offenses detected

**Comment by Darkhost1999** [comment]
https://github.com/RPCSX/rpcsx/commit/7e7a56f0e2c96381140c149581b0bbc1a0503001

I can update running.md with arguments simplification. Or should I add the information as basic/advanced commands?

File: install-guide.md [hunk 1]
```diff
@@ -25,51 +25,46 @@ We provide two installation methods:
...
 ```
+
 go to [Continued Install.](/wiki/installation/#continued-install)
 
   - **Arch**
     - vulkan-devel is a group, and you should install **all**!
+
 ```sh
-sudo pacman -S --needed git base-devel libunwind glfw-x11 vulkan-devel
+sudo pacman -S libunwind glfw-x11 vulkan-devel sox git cmake alsa-lib nasm
```
Comment (OCDkirby) [inline install-guide.md]: `--needed` should be preserved

**Review by OCDkirby — CHANGES_REQUESTED** [review]

**Comment by OCDkirby** [comment]
> https://github.com/RPCSX/rpcsx/commit/7e7a56f0e2c96381140c149581b0bbc1a0503001
> 
> I can update running.md with arguments simplification. Or should I add the information as basic/advanced commands?

Maybe add it as an optional argument under see the relevant command?

**Comment by Darkhost1999** [comment]
I'm going to sleep. Maybe I'll add something like this in the morning.
```
##Booting Firmware
Standard Syntax
./rpcsx --system --mount /path/to/firmware/md0/ / /mini-syscore.elf

Simplified Syntax
./rpcsx --fw /path/to/firmware

##Booting Games
Standard Syntax
./rpcsx --mount /path/to/firmware/md0/ / --mount /path/to/game/app0 /app0 /app0/eboot.bin [optional args without brackets]

Simplified Syntax
./rpcsx --fw /path/to/firmware /path/to/game/

For most samples, you still need to specify mount points, e.g.
--fw /path/to/firmware--mount /path/to/api_gnm/directory/ /app0 /app0/basic-sample/basic-sample_debug.elf
```

**Comment by Darkhost1999** [comment]
Seems unnecessary. They can ask on Discord if syntax can be simplified but nothing wrong with being more detailed or specific about what you want to load. Especially at this time. Maybe simplify the context at 12% compatibility database for 5k games listed.

**Review by OCDkirby — APPROVED** [review]

**Pull request merged**